src/Raven.Client/Documents/Operations/AI/AiUsage.cs

```diff
@@ -42,7 +50,7 @@ internal static AiUsage GetUsageDifference(AiUsage usage2, AiUsage usage1)
     {
         return new AiUsage
         {
-            PromptTokens = usage2.PromptTokens - usage1.PromptTokens,
+            PromptTokens = usage2.PromptTokens - usage1.TotalTokens,
```

Comment:
Still need protection from going negative, no?
What is the previous `TotalTokens` was before summary, and the one is afterward?

I need to be something like: 

```
TotalTokens = Math.Min(usage2.TotalTokens - usage1.TotalTokens, usage2.TotalTokens),
```